packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -462,24 +474,7 @@ class RenderParagraph extends RenderBox
       }
       child = childAfter(child);
     }
-    return false;
-  }
-
-  @override
-  void handleEvent(PointerEvent event, BoxHitTestEntry entry) {
```

Comment:
Is this logic still needed anywhere? I see it was removed without being added anywhere else.

Reply:
Ahh, nevermind, found the corresponding handler above.